README.md

```
@@ -91,6 +91,28 @@ A dump of addresslib for SkyrimSSE 1.6.318.0 (AE). This should be considered can
 * aeid  - Addresslib id for Anniversary Edition
 * ae_addr - AE offset (e.g., 140001022)
 
+##### offsets-1-6-1170.0.csv
+
+A dump of addresslib for SkyrimSSE 1.6.1170.0 (AE). This should be considered canonical for the aeid -> ae_addr mapping.
+
+|aeid|ae_addr|
+|--|--|
+1|1022
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)